runtime/westend/src/weights/xcm/pallet_xcm_benchmarks_generic.rs

```diff
@@ -51,57 +51,57 @@ impl<T: frame_system::Config> WeightInfo<T> {
 	// Storage: Dmp DownwardMessageQueueHeads (r:1 w:1)
 	// Storage: Dmp DownwardMessageQueues (r:1 w:1)
 	pub(crate) fn report_holding() -> Weight {
-		(35_446_000 as Weight)
+		(24_000_000 as Weight)
 			.saturating_add(T::DbWeight::get().reads(5 as Weight))
 			.saturating_add(T::DbWeight::get().writes(3 as Weight))
 	}
 	pub(crate) fn buy_execution() -> Weight {
-		(5_228_000 as Weight)
+		(3_377_000 as Weight)
 	}
 	// Storage: XcmPallet Queries (r:1 w:0)
 	pub(crate) fn query_response() -> Weight {
-		(18_708_000 as Weight)
+		(12_418_000 as Weight)
 			.saturating_add(T::DbWeight::get().reads(1 as Weight))
 	}
 	pub(crate) fn transact() -> Weight {
-		(20_401_000 as Weight)
+		(12_795_000 as Weight)
 	}
 	pub(crate) fn refund_surplus() -> Weight {
-		(5_238_000 as Weight)
+		(3_485_000 as Weight)
 	}
 	pub(crate) fn set_error_handler() -> Weight {
-		(5_104_000 as Weight)
+		(3_314_000 as Weight)
 	}
 	pub(crate) fn set_appendix() -> Weight {
-		(5_095_000 as Weight)
+		(3_334_000 as Weight)
```

Comment:
33% reduction in weights?

Reply:
Huh, could it be related to the increased DB cache size? I can bench again to see if we get anything different.

Reply:
Weights in the Polkadot repo were not updated after adding the Production optimizations in https://github.com/paritytech/substrate/pull/10692.  
So this should happen for other runtimes and pallets as well.